Projects/Multi/Applications/LoRa/DRAGINO-LRWAN(AT)/inc/version.h

```diff
@@ -73,6 +73,7 @@ Maintainer: Miguel Luis, Gregory Cristian and Wael Guibene
 #define LRWAN_VERSION  (uint32_t) 0x00001140  /*3 next hex is i_cube release*/
 #define VERSION   (uint32_t) ( LORA_MAC_VERSION | LRWAN_VERSION | TEST_VERSION )
 #define AT_VERSION_STRING	"v1.6.7"
+#define FIRMWARE_VERSION_MINOR	7
```

Comment:
It's actually not the minor version but the _patch_ version.

Reply:
true dat! I will update the patch.

Your other suggestions make sense too, they are a little more work but better in the long term.